src/main/java/de/tum/cit/aet/artemis/iris/web/IrisSettingsResource.java

```diff
@@ -89,7 +89,7 @@ public ResponseEntity<IrisSettings> getRawCourseSettings(@PathVariable Long cour
      * @return the {@link ResponseEntity} with status {@code 200 (Ok)} and with body the settings, or with status {@code 404 (Not Found)} if the exercise could not be found.
      */
     @GetMapping("exercises/{exerciseId}/raw-iris-settings")
-    @EnforceAtLeastEditorInExercise
+    @EnforceAtLeastTutorInExercise
     public ResponseEntity<IrisSettings> getRawExerciseSettings(@PathVariable Long exerciseId) {
```

Comment:
The `authCheckService.checkHasAtLeastRoleForExerciseElseThrow(Role.STUDENT, exercise, user);` can be removed in this method.

Reply:
Tank you for the review! I removed it.